app/code/Vladimirl/Chatter/Model/ResourceModel/ChatMessage.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+
+namespace Vladimirl\Chatter\Model\ResourceModel;
+
+use Magento\Framework\Model\ResourceModel\Db\AbstractDb;
+
+class ChatMessage extends AbstractDb
```

Comment:
@Vladimir-L Use full class names in `extends`, `implements`, in constructor and in the property definition.